packages/did-provider-ebsi/src/services/EbsiRPCService.ts

```diff
@@ -0,0 +1,157 @@
+import { Headers } from 'cross-fetch'
+import {
+  AddVerificationMethodParams,
+  AddVerificationMethodRelationshipParams,
+  GetDidDocumentParams,
+  GetDidDocumentsParams,
+  InsertDidDocumentParams,
+  SendSignedTransactionParams,
+  UpdateBaseDocumentParams,
+  Response,
+  GetDidDocumentsResponse,
+} from '../types'
+import { DIDDocument } from 'did-resolver'
+
+/**
+ * @constant {string} jsonrpc
+ */
+const jsonrpc = '2.0' // optional param of plugin?
+const baseUrl = 'https://api-pilot.ebsi.eu/did-registry/v5/jsonrpc' // optional param of plugin?
+const token = '' // optional param of plugin?
```

Comment:
Why is this const listed in a file containing functions. This should be an argument to each function and the plugin should be responsible for fetching (and caching) the token

Reply:
Made it an argument to each function